scripts/templates/browser-bookmarklet.template.js

```diff
@@ -1,190 +1,49 @@
-const snippet = (name, integrity, version, globalVar, apiKey, userId, serverZone) => `
+const snippet = (
+  name,
+  integrity,
+  version,
+  globalVar,
+  apiKey,
+  userId,
+  serverZone,
+  ingestionSourceName,
+  ingestionSourceVersion,
+  autoTrackingPluginVersion,
+) => `
 !(function (window, document) {
   var amplitude = window.${globalVar} || { _q: [], _iq: {} };
   if (amplitude.invoked) window.console && console.error && console.error('Amplitude snippet has been loaded.');
   else {
     amplitude.invoked = true;
+    var s = document.getElementsByTagName('script')[0];
+    var autoTrackingPluginScript = document.createElement('script');
+    autoTrackingPluginScript.src = 'https://cdn.amplitude.com/libs/plugin-auto-tracking-browser-${autoTrackingPluginVersion}-min.js.gz';
+    autoTrackingPluginScript.async = false;
+    s.parentNode.insertBefore(autoTrackingPluginScript, s);
     var as = document.createElement('script');
     as.type = 'text/javascript';
     as.integrity = '${integrity}';
     as.crossOrigin = 'anonymous';
-    as.async = true;
+    as.async = false;
```

Comment:
why this need to be false?

Reply:
This will make sure the scripts are loading in order, so when we init the SDK, it has the auto-tracking plugin ready to use. For more information, check: https://jason-ge.medium.com/dynamically-load-javascript-files-in-order-5318ac6bcc61.